src/contrast/pages/UploadNewPaper/ExperimentsSection/ExperimentForm.jsx

```diff
@@ -138,6 +141,13 @@ export default function ExperimentForm({
       label: rawTextToShow(type),
     }));
   const shouldCheckAllClassificationsFilled = false;
+
+  if (experimentData) {
```

Comment:
This is not good. A hook is not supposed to be inside a conditional:

see [this](https://legacy.reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level)

Reply:
This is not fixed yet